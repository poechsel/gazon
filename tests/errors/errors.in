login u1
pass p1
ls
cd ../../../mnt
ls
put looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong 200
rm ../..
mkdir looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong
