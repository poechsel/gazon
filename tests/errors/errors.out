total 4
<*> 1 <*> <*> 136 <*> server_file.txt
Error: access denied.
total 4
<*> 1 <*> <*> 136 <*> server_file.txt
Error: the path is too long.
Error: access denied.
Error: the path is too long.
